RNTester/js/RNTesterApp.ios.js

```diff
@@ -34,6 +33,14 @@ const {
   YellowBox,
 } = require('react-native');
 
+// [TODO(macOS ISS#2323203)
+const RNTesterList =
+  Platform.OS === 'macos'
+    ? /* $FlowFixMe allow iOS to list macOS file */
+      require('./utils/RNTesterList.macos')
```

Comment:
Can't you just change the import to `require('./utils/RNTesterList')` and the bundler should pick between the ios/macos versions for you?

Reply:
To be honest I'm not sure, a lot of time have passed since creation of this PR. If I remember correctly this tweak was required to fix the CI.